### DISCUSSION TITLE
Unable to locate a class or view for component [dashboard].

### DISCUSSION THREAD

**Discussion by imhugos** [spatie/laravel-dashboard#77] [Q&A]
I'm using the Spatie Laravel Dashboard in Laravel Framework 8.49.0 and in my local machine (MacOS Big Sur / MAMP Pro / PHP 7.4) everything is working fine, but when I deployed to a production machine I get the error "**Unable to locate a class or view for component [dashboard]**." 

I'm following the instructions in [https://spatie.be/docs/laravel-dashboard/v2/basic-usage/creating-your-first-dashboard](https://spatie.be/docs/laravel-dashboard/v2/basic-usage/creating-your-first-dashboard) and have several custom made tiles. I believe both local and production systems are case sensitive, so I have no idea what the problem can be.

I'm trying to fix this for days with no success. Can you please help?

**Comment by maxime-anoloc** [comment]
Same issue here, Laravel 11.44 / PHP 8.3.30 but in dev env.

**Reply by maxime-anoloc** [reply]
@imhugos Found the issue. In my case, in the project I'm working on, the Blade service containers were overridxen in an bogus manner (the goal was to add custom Blade directives). Since it's a pretty old Laravel project, extending Blade was made using an old-school fashion.
